libs/model/src/community/UpdateCommunity.command.sample.ts

```diff
@@ -1,14 +1,13 @@
 import { type Command } from '@hicommonwealth/core';
```

Comment:
I don't think we need samples anymore. Let's get rid of the samples.

Reply:
will remove in next PR